core/layout_api.php

```diff
@@ -1262,3 +1215,54 @@ function layout_login_page_logo() {
 	</div>
 	<?php
 }
+
+/**
+ * Returns a single link for the "manage" menu item in sidebar, based on current
+ * user permissions, and priority if several subpages are available.
+ * If there is not any accesible manage page, returns null.
+ * @return string|null	Page name for the manage menu link, or null if unavailable.
+ */
+function layout_manage_menu_link() {
+	static $t_link = null;
+	if( access_has_global_level( config_get( 'manage_site_threshold' ) ) ) {
+		$t_link = 'manage_overview_page.php';
+	} else {
+		if( access_has_global_level( config_get( 'manage_user_threshold' ) ) ) {
+			$t_link = 'manage_user_page.php';
+		} else {
+			if( access_has_any_project_level( 'manage_project_threshold' ) ) {
+				if( $t_current_project == ALL_PROJECTS ) {
```

Comment:
$t_current_project is not defined at this place

Reply:
fixed